app/models/item.rb

```diff
@@ -3,4 +3,15 @@ class Item < ActiveRecord::Base
   validates :list_id, presence: true
   validates :name, presence: true
 
+  def date
+    if @item.due_date = Date.today
+      return 'Today'
+    elsif @item.due_date = Date.tomorrow
+      return 'Tomorrow'
+    else @item.due_date = Date.tomorrow + 1
+      return 'Day After Tomorrow'
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.